crates/rust-analyzer/Cargo.toml

```
@@ -23,7 +23,7 @@ crossbeam-channel = "0.5.5"
 dissimilar = "1.0.4"
 itertools = "0.10.5"
 scip = "0.1.1"
-lsp-types = { version = "0.93.1", features = ["proposed"] }
+lsp-types = { version = "0.93.2", features = ["proposed"] }
```

```suggestion
lsp-types = { version = "=0.93.2", features = ["proposed"] }
```
We might wanna pin our lsp-types version to prevent breakages like these in the future

---

This was `lsp-types` breaking semver in a minor release, but sure, it can't hurt. I just hope we don't end up doing this for every dependency.

---

Oh for some reason I thought they were breaking by keeping versions numbers in sync, but thats not the case. Nevermind then, that's just unfortunate i guess

---

It's fine to pin it, I think it's not the first time something like this happens. But if rather not use `default()` so that we know when something new gets added there.

---

https://github.com/gluon-lang/lsp-types/pull/248#issuecomment-1295018719 so ye, pinning is the right choice for us here